### DISCUSSION TITLE
重复活动不好用啊

### DISCUSSION THREAD

**Discussion by SeinaU** [KIvoy/nikke-assistant#4] [Q&A]
Repeating event level started...
Could not find home_event_chainsaw_2023_feb_step_1
could not repeat event levels
Exiting to home...
Exited to home succesfully

在第一步就卡住了，更新了step的截图也没反应

**Comment by KIvoy** [comment]
@SeinaU， 这个有点奇怪， 请问他是不是点开了活动， 然后自动鼠标下滑了呢？ 从log上来看是没能找到第一步的图， 你能完全肉眼看见全部的第一步的截图么？
 如果没试过可以试一下：
1. 重开游戏
2. 点重新加载
3. 然后再试下

**Comment by SeinaU** [comment]
@KIvoy 
你好，感谢回复。
他是点开了活动图标， 然后自动鼠标下滑到了能完整看到活动banner的地方，然后就开始出错了说找不到
step_1默认的图和自己的截图都试过了

> Could not find home_event_chainsaw_2023_feb_step_1
> could not repeat event levels

用了重新加载的方法也不行。。

**Reply by KIvoy** [reply]
@SeinaU 我分析一下你试试对不对， 既然他自己点了活动图标说明OCR没有太大的问题， 那估计就是分辨率/画质的问题导致不能识别。我看了下我的code发现这部分因为准确性需要0.99相同， 所以很有可能是截图和你游戏画质不一致， 或者是截图部分大小有误。 你可以试着把游戏调到最高画质， 然后用原文件试一试。 如果不行自己再截图试试。 要是实在不行可以等我明天更新一下这个部分。
